src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -154,56 +154,45 @@ interface ExtensionSuggestion {
 	isExtensionPack?: boolean;
 	icon?: string;
 	link?: string;
-	extensionPackExtensions?: any[];
 }
 
 
+interface ExtensionPackExtensions {
+	name: string;
+	icon: string;
+}
+
 const extensionPacks: ExtensionSuggestion[] = [
 	{
 		name: localize('welcomePage.adminPack', "SQL Admin Pack"),
 		title: localize('welcomePage.showAdminPack', "SQL Admin Pack"),
 		description: localize('welcomePage.adminPackDescription', "Admin Pack for SQL Server is a collection of popular database administration extensions to help you manage SQL Server"),
 		id: 'microsoft.admin-pack',
-		extensionPackExtensions: [
-			{
-				name: 'SQL Server Agent',
-				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-agent-extension?view=sql-server-2017',
-				icon: '../../../../../extensions/admin-pack/images/extension.png'
-			},
-			{
-				name: 'SQL Server Profiler',
-				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-profiler-extension?view=sql-server-2017',
-				icon: '../../../../../extensions/admin-pack/images/extension.png'
-			},
-			{
-				name: 'SQL Server Import',
-				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-import-extension?view=sql-server-2017',
-				icon: '../../../../../extensions/admin-pack/images/extension.png'
-			},
-			{
-				name: 'SQL Server Dacpac',
-				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-dacpac-extension?view=sql-server-2017',
-				icon: '../../../../../extensions/admin-pack/images/extension.png'
-			}
-		],
 		isExtensionPack: true
 	},
 ];
 
+const extensionPackExtensions: ExtensionPackExtensions[] = [
+	{ name: 'SQL Server Agent', icon: '../../../../sql/workbench/contrib/welcome/defaultExtensionIcon.svg' },
```

Comment:
Pretty sure this is still going to be broken on packaged builds since the path location changes when we webpack. This should work though : `require.toUrl('./media/defaultExtensionIcon.svg');`

(as part of this I would also suggest moving the icons into a media folder in this folder - it's the practice we use elsewhere)